### PR TITLE
Feature/179 180 182 187 comments actvity feed

### PR DESCRIPTION
**Pull request by chris-lawton**
contains FE work for comments styling and activity feed

### CODE REVIEW

File: opentech/static_src/src/sass/apply/components/_activity-feed.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        background: $color--white;
+        opacity: 1;
+    }
+
+    &__header {
+        padding: 0 20px;
+        background-color: $color--default;
+
+        @include media-query(tablet-portrait) {
+            padding: 0;
```
Comment (martycoote) [inline opentech/static_src/src/sass/apply/components/_activity-feed.scss]: This padding removal means the close icon is right up against the window edge until the wrapper--medium max-width applies at 1020px. screen grab attached.

otf-dev torchboxapps com_apply_submissions_

Comment (chris-lawton) [reply opentech/static_src/src/sass/apply/components/_activity-feed.scss]: Good spot - addressed here: https://github.com/OpenTechFund/opentech.fund/pull/135/commits/886dc8015b458ea87bb0d14075e50995d1eeb4bc

File: opentech/apply/activity/templates/activity/include/listing_base.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
         <div class="feed__heading">
             <h4 class="feed__name">{{ activity.user }}</h4>
             {% if submission_title %}
-                <h6>for</h6>
+                <h6>udpated</h6>
                 <h6><a href="{{ activity.submission.get_absolute_url }}">{{ activity.submission.title }}</a></h6>
```
Comment (zerolab) [inline opentech/apply/activity/templates/activity/include/listing_base.html]: should this not be a single h6 tag?

Comment (zerolab) [reply opentech/apply/activity/templates/activity/include/listing_base.html]: also, udpated -> updated

**Pull request merged**